usmqe_tests/api/grafana/test_cluster_dashboard.py

```diff
@@ -74,6 +74,8 @@ def test_cluster_dashboard_layout():
             'IOPS',
             'Capacity Utilization',
             'Capacity Available',
+            'Weekly Growth Rate',
+            'Weeks Remaining',
```

Comment:
Why did you add 'Weekly Growth Rate' and 'Weeks Remaining'? The test fails now on the latest build.

Reply:
This was a typo, thanks for noticing. I will fix it